static/js/echo.js

```diff
@@ -184,7 +184,7 @@ exports.reify_message_id = function reify_message_id(local_id, server_id) {
     message.id = server_id;
     message.locally_echoed = false;
 
-    var opts = {old_id: local_id, new_id: server_id};
+    var opts = {old_id: +local_id, new_id: server_id};
```

Comment:
I might prefer to do something more visible for casting a str to int here.

Reply:
Probably just `parseInt`?  Or we could do a `local_id_int = `... thing.

Reply:
To be clear, it’s a float. Updated with `parseFloat`.